README.md

```diff
@@ -63,6 +65,15 @@ Use `plug.set_path` to set a custom install path for plugins.
 plug.set_path('/Users/user/my-plugins')
 ```
 
+### Themes
+
+Themes are installed under `plug.set_path/themes/pluginname` 
+Which means to use them you have to include in your visrc 
+
```

Comment:
Can you change this to: 
"
Themes are installed to `plug.set_path/themes/plugin_name`.  In order to use them, add the following to your `visrc`file:
"